src/datatable/xlsx.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+#-------------------------------------------------------------------------------
+# Copyright 2018-2021 H2O.ai
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#-------------------------------------------------------------------------------
+import datatable as dt
+
+
+
+def read_xlsx_workbook(filename, subpath):
+    try:
+        import openpyxl
+    except ImportError:
+        raise dt.exceptions.ImportError(
+            "Module `openpyxl` is required in order to read Excel file '%s'"
+            % filename)
+
+    if subpath:
+        wb = openpyxl.load_workbook(filename, data_only=True)
+        range2d = None
+        if subpath in wb.get_sheet_names():
```

Comment:
So the tests are failing because `.get_sheet_names()` is deprecated, and `.sheetnames` should be used instead: https://openpyxl.readthedocs.io/en/stable/api/openpyxl.workbook.workbook.html#openpyxl.workbook.workbook.Workbook.get_sheet_names

Reply:
Now it is `DeprecationWarning: Call to deprecated function get_sheet_by_name (Use wb[sheetname])`. Feel free to modify `appveyor.yml` by adding `openpyxl` to the `pip install ...` command, then you should see the same errors on AppVeyor.